test/e2e/util.go

```diff
@@ -1721,6 +1722,14 @@ func getNodeEvents(c *client.Client, nodeName string) []api.Event {
 	return events.Items
 }
 
+func ListSchedulabeNodesOrDie(c *client.Client) *api.NodeList {
```

Comment:
I think that we should return error from this method too and handle it individually it the code.
In some cases, we may want to do expectNoError(), but it may happen that we may want to retry on failure.

So I really vote for returning an error here.